pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[project]
```

Comment:
If you use `pyproject.toml`, I think it makes sense to fully migrate the build system and remove `requirements.txt` etc.

Check the `main` branch of the TESK repo for an example. You can also ask @JaeAeich for help.

Reply:
remove requirement and add the deps as
```bash
poetry add crypt4gh>=1.7
```
and similarly it would be for dev deps
```bash
poetry add <package_name> --group=dev
```
I think it would be a good idea to look at this [generic pyproject.toml](https://github.com/elixir-cloud-aai/cookiecutter-python/blob/dev/%7B%7B%20cookiecutter.project_name%20%7D%7D/pyproject.toml).